calyx-py/calyx/utils.py

```diff
@@ -2,16 +2,22 @@
 import math
 
 
-def block(decl, contents, indent=2, sep="\n"):
+def block(decl, contents, indent=2, sep="\n", metadata=False):
     """Format a block like this:
         decl {
           contents[0] <sep> contents[1] ...
         }
     where `decl` is one line and contents is a list that is separated with `sep`.
+    If `metadata` is `True` then the block would be formatted without curly braces.
```

Comment:
can you rename this to `with_curly` or similar?

Reply:
done!